cmake/legacy.cmake

```diff
@@ -158,7 +158,7 @@ if(ICU_ROOT)
   set(dds_icu_hint "-DDDS_LD_LIBRARY_PATH=${ICU_ROOT}/lib")
 endif()
 list(APPEND packages dds)
-set(dds_version "3.6")
+set(dds_version "3.7.22")
```

Comment:
When we had issues in the past, @AnarManafov adviced against distributing development versions of DDS. Has this changed? Why do you want this new DDS?

Reply:
I simply updated to the latest available version, but i am also fine with 3.6 or 3.7 if uneven numbers doesn't mean development versions.

Reply:
My latest understanding is that 3.7 (uneven minor version) is development.

Reply:
Went back to version 3.6

Reply:
@dennisklein , @fuhlig1, yes this is a developer version. But last couple of years, working with ALICE, I had to release dev versions pretty frequently. According to their policy a patch (dev) version has a shorter release path in comparison to a new major release. I therefor was using this trick and have been releasing only dev versions.  

You can add this one. It is stable.

Reply:
@dennisklein,

should we bump to the latest version if it is stable?

Reply:
> should we bump to the latest version if it is stable?

yes, fine with me then :)

Reply:
@fuhlig1 , yeah, anything tagged was used on  ALICE prod.

Reply:
Bumped DDS to 3.7.10